BrainPortal/app/views/nh_messages/new.html.erb

```diff
@@ -0,0 +1,62 @@
+
+<%-
+#
+# Neurohub Project
+#
+# Copyright (C) 2019-2020
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+-%>
+
+<% title 'New Message', "" %>
+<div class="nh_content">
+  <div class="nh_form">
+    <%= error_messages_for @message, :header_message => "Message could not be send." %>
+    <p class="nh_form_title">Send Message</p>
+
+    <%= form_for @message, :as => :message, :url => {:action => "create"}, :datatype => "script" do |f| -%>
+
+      <fieldset>
+
+        <%= f.label :destination_group_id,
+            "To a user or a group (project members):" %>
+        <%= group_select( :destination_group_id, # does not goes into the model
+                          {
+                            :groups   => @recipients,
+                            :selector => @destination_group_id, # :group_id is not an attribute of Message.
+                          },
+                          {:include_blank => true}
+            )
+         %>
+      </fieldset>
+      <fieldset>
+        <%= f.label :header %>
+        <%= f.text_field :header, :size => 92 %>
```

Comment:
put a limit on the size of the content too, I get an exception if I put too much text in the header or body